### PR TITLE
docs: install options -> uninstall options

### PR DESCRIPTION
**Pull request by DriesDeMaeseneer**
### What does this PR try to resolve?

Fixed a typo in the cargo-uninstall.md documentation file.

### CODE REVIEW

File: src/doc/src/commands/cargo-uninstall.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ The installation root is determined, in order of precedence:
 
 ## OPTIONS
 
-### Install Options
+### Uninstall Options
```
Comment (weihanglo) [inline src/doc/src/commands/cargo-uninstall.md:30]: Thank you for your contribution!

This markdown file is generated. You'll need to update the manpage template instead: https://github.com/rust-lang/cargo/blob/82c489f1c612096c2dffc48a4091d21af4b8e046/src/doc/man/cargo-uninstall.md?plain=1#L24

See also the doc contribution guide: https://github.com/rust-lang/cargo/tree/master/src/doc#building-the-man-pages

Comment (DriesDeMaeseneer) [reply src/doc/src/commands/cargo-uninstall.md]: Thanks for the feedback! 

I moved the typo to the correct file.

Comment (weihanglo) [reply src/doc/src/commands/cargo-uninstall.md:30]: Thanks! Could you squash commits into one? We would like to keep every commit passing CI if possible.

Comment (DriesDeMaeseneer) [reply src/doc/src/commands/cargo-uninstall.md:30]: I will! Thanks for your patience!

Comment (DriesDeMaeseneer) [reply src/doc/src/commands/cargo-uninstall.md:30]: Hey @weihanglo, is there any way for me to stop the CI when I know that it's going to fail? It seems wasteful to run the full pipeline in that case.

Comment (weihanglo) [reply src/doc/src/commands/cargo-uninstall.md:30]: Push a new rev and the old one will get cancelled.

https://github.com/rust-lang/cargo/blob/82c489f1c612096c2dffc48a4091d21af4b8e046/.github/workflows/main.yml#L19-L21

**Review by weihanglo — APPROVED** [review]

**Comment by weihanglo** [comment]
@bors r+

Thank you!

**Comment by weihanglo** [comment]
Has docker rate limit recovered?

@bors retry

**Pull request merged**